source/channels/make-calls.rst

```diff
@@ -33,13 +36,22 @@ You can share a call's link to use in a meeting request or share with other team
 
 The call link is valid for long as the channel is active. When a channel is archived or deleted the link will become invalid.
 
+Record calls
+------------
+
+The host of a meeting can record the call. The recording includes audio as well as any screen sharing during the call. The default setting for a recording is 60 minutes, but your system admin may adjust that as needed. Ten minutes before the time limit, you'll receive a reminder that the recording limit will be reached soon.
+
+If your call is going to continue and you still want to record, once the first recording is complete you can start a new one immediately after.
+
+You can access the call recording locally.
```

Comment:
@streamer45, I'm pretty sure this is an incorrect assumption. What's the process for accessing the recording?

Reply:
Under normal circumstances (everything working as expected that is) call recordings are posted by the Calls bot directly in the call thread as normal file attachment. That's how users will access recordings for a given call.

Reply:
Brilliant, thank you!
(This is still a WIP but wanted to get these out of the way). :)